docs/index.md

```diff
@@ -1,38 +1,34 @@
-## Explore, train and deploy your deep learning projects using FloydHub
+## Introduction
+Welcome to [FloydHub](https://www.floydhub.com/)! Here you'll find comprehensive information for training and deploying your deep learning and AI applications with our platform. We've tried to make this documentation user-friendly and example-filled, but if you have any questions, please visit the [community forum](https://forum.floydhub.com/) or [contact us](mailto:support@floydhub.com).
 
+The fastest way to get up and running is to use our [quickstart guide](http://docs.floydhub.com/getstarted/quick_start), which walks through an entire FloydHub training job step-by-step. You'll create a new Project on the FloydHub web dashboard, connect it to a local directory on your computer, and then kick-off a job using the FloydHub CLI to train your deep learning model on FloydHub's GPU servers.
 
-### What is FloydHub?
-[FloydHub](https://www.floydhub.com/) is a platform for data scientists building deep learning and AI applications. 
+## Deep learning without the DevOps:
 
-Using FloydHub, you can:
+### Frictionless data science
+Why worry about provisioning GPUs, installing drivers, or managing software dependency hell? With FloydHub, we take care of your entire deep learning DevOps workflow - so you can focus on the science.
 
-- Explore popular deep learning projects and datasets
-- Train your *own* deep learning models on the cloud without *any* DevOps
-- Share and collaborate on your projects
-- Deploy your models as REST endpoints
+Training a PyTorch model using GPUs on the cloud is as simple as executing this command on your terminal: 
+`floyd run --gpu --env pytorch "python train.py"`. Try it now with our [quickstart guide](http://docs.floydhub.com/getstarted/quick_start).
```

Comment:
quick starts uses tensorflow, maybe change the sample command to `floyd run --gpu --env tensorflow "python train.py"`?